source/features/forked-to.tsx

```diff
@@ -51,12 +43,17 @@ async function updateUI(forks: string[]): Promise<void> {
 		return;
 	}
 
+	const url = new GitHubURL(location.href);
+	if (url.branch) {
```

Comment:
This should solve that issue
```suggestion
	if (!pageDetect.isRepoRoot() && (pageDetect.isSingleFile() || pageDetect.isRepoTree())) {
```

Reply:
It needed an else.